content/vendor/packaging/private-images.md

```diff
@@ -64,3 +63,7 @@ This secret is based on the customer license, and will be used to pull all image
 
 Images hosted at `registry.replicated.com` will not be rewritten.  
 However, the same secret will be added to those PodSpecs as well.
+
+## Additional namespaces
+
+When deploying pods to namespaces other than the KOTS application namespace, the namespace must be added to the `additionalNamespaces` attribute of the [Application](/reference/v1beta1/application/) spec. This will ensure that an imagePullSecret named `kotsadm-replicated-registry` exists in the namespace to allow the pod to pull the image. For more information about the `additionalNamespaces` attribute see [this doc](/vendor/operators/additional-namespaces/).
```

Comment:
When deploying pods to namespaces other than the KOTS application namespace, the namespace must be added to the `additionalNamespaces` attribute of the [Application](/reference/v1beta1/application/) spec _**(usually in the replicated_app.yaml).**_ This will ensure that an imagePullSecret named `kotsadm-replicated-registry` **_gets auto-provisioned (by kots)_** in the namespace to allow the pod to pull the image. For more information about the `additionalNamespaces` attribute see [this doc](/vendor/operators/additional-namespaces/).